spring-cloud-dataflow-server-core/src/main/java/org/springframework/cloud/dataflow/server/stream/SkipperStreamDeployer.java

```diff
@@ -352,7 +351,7 @@ public AppStatus getAppStatus(String id) {
 	@Override
 	public RuntimeEnvironmentInfo environmentInfo() {
 		// TODO To fix when https://github.com/spring-cloud/spring-cloud-skipper/issues/392
-		throw new IncompatibleStreamDeployerException("AppDeployer");
+		throw new UnsupportedOperationException("EnvironmentInfo is not supported for Skipper yet (issue#392)");
```

Comment:
Having Skipper in the message is good. I think `(issue#392)` can be removed as the comment there is sufficient.

Reply:
+1